resource-watch/resource-watch
# Fix: set the default layer on open in map click

## Overview
This PR fixes the selection of default layer when user click Open in Map button.
## Testing instructions
Go to http://localhost:9000/data/explore
Select a dataset with multiple layers. Go to details page. Click open in map.You will see the default layer.
## [Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/169711432)

---

## Checklist before submitting
